perses/annihilation/new_relative.py

```diff
@@ -55,6 +55,10 @@ def __init__(self, topology_proposal, current_positions, new_positions, use_disp
             The positions of the "new system"
         use_dispersion_correction : bool, default False
             Whether to use the long range correction in the custom sterics force. This is very expensive for NCMC.
+        functions : dict, default None
+            Alchemical functions that determine how each force is scaled with lambda. The keys must be strings with
+            names beginning with lambda_ and ending with each of bonds, angles, torsions, sterics, electrostatics.
+            If none, then the integrator will need to set each of these and parameter derivatives will be unavailable.
```

Comment:
Do you always have to provide all lambda functions, and set the ones you don't use to `None`, or only the ones you use? Not sure what `if none` indicates.